<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPPTest</class>
 <widget class="QWidget" name="SPPTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_spp">
     <property name="text">
      <string>Connecting to remote device...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="msg_spp"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="chat_spp">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bt_spp_done">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>msg_spp</sender>
   <signal>returnPressed()</signal>
   <receiver>SPPTest</receiver>
   <slot>msgWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_spp_done</sender>
   <signal>clicked()</signal>
   <receiver>SPPTest</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>msgWrite()</slot>
  <slot>done()</slot>
 </slots>
</ui>
